src/test/java/ch/wisv/areafiftylan/OrderRestIntegrationTest.java

```diff
@@ -116,959 +80,544 @@ public void testViewAllOrders_User() {
     }
 
     @Test
-    public void testViewAllOrders_Admin() {
-        insertTestOrders();
-
-        SessionData login = login(admin.getUsername(), adminCleartextPassword);
+    public void testGetAllOrdersAsUser() {
+        insertAnonOrder();
+        User user = createUser();
 
         //@formatter:off
         given().
-            filter(sessionFilter).
-            header(login.getCsrfHeader()).
+            header(getXAuthTokenHeaderForUser(user)).
         when().
             get(ORDER_ENDPOINT).
         then().
-            statusCode(HttpStatus.SC_OK).
-            body("$", hasSize(2)).
-            body("user.username", containsInAnyOrder(admin.getUsername(), user.getUsername())).
-            body("status", hasItems("ASSIGNED", "ASSIGNED"));
-        //formatter:on
+            statusCode(HttpStatus.SC_FORBIDDEN).body("message", containsString("denied"));
+        //@formatter:on
     }
 
-    //     @RequestMapping(value = "/orders", method = RequestMethod.POST)
-
     @Test
-    public void testCreateSingleOrder_User() {
-        Map<String, String> order = new HashMap<>();
-        order.put("pickupService", "true");
-        order.put("type", TicketType.TEST.toString());
-        order.put("chMember", "false");
-        SessionData login = login(user.getUsername(), userCleartextPassword);
+    public void testGetAllOrdersAsAdmin() {
+        insertAnonOrder();
+        User admin = createUser(true);
 
         //@formatter:off
         given().
-            filter(sessionFilter).
-            header(login.getCsrfHeader()).
+            header(getXAuthTokenHeaderForUser(admin)).
         when().
-            content(order).contentType(ContentType.JSON).
-            post(ORDER_ENDPOINT)
-        .then().
-            statusCode(HttpStatus.SC_CREATED).
-            body("object.user", is(nullValue())).
-            body("object.status", is("ANONYMOUS")).
-            body("object.tickets", hasSize(1)).
-            body("object.tickets.pickupService", hasItem(true)).
-            body("object.tickets.type", hasItem(is(TicketType.TEST.toString()))).
-            body("object.amount",equalTo(TicketType.TEST.getPrice() + TicketOption.PICKUPSERVICE.getPrice()));
+            get(ORDER_ENDPOINT).
+        then().
+            statusCode(HttpStatus.SC_OK).
+            body("$", hasSize(Long.valueOf(orderRepository.count()).intValue()));
```

Comment:
The archived expired order are in a completely different repo, and the 30 minutes they're expired as an Order, it's not unwanted to have them here.